src/config/ConfigBuilder.ts

```diff
@@ -181,6 +191,11 @@ export class ConfigBuilder {
     }
 
     private handleSSL(jsonObject: any): void {
+        for (const key in jsonObject) {
+            if (!['enabled', 'sslOptions', 'sslOptionsFactory', 'sslOptionsFactoryProperties'].includes(key)) {
```

Comment:
I think it would be good to either use SSLConfig keys here or somehow define this and reuse it, instead of such usage